app/templates/CommentList.rss.twig

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<rss version="2.0">
+    <channel>
+        <title>Spenden an Wikimedia</title>
+        <link>{$ rssLink |e $}</link>
```

Comment:
I'm sure. I've added a test in #80 and removed the `|e` and it still passed. But keep them if you like.